frontend/src/app/shared/services/countdown.service.ts

```diff
@@ -18,15 +18,18 @@ export class CountdownService {
           new Date(now.getFullYear(), now.getMonth(), now.getDate(), hour, minute, 0),
           ts => {
             // if minutes or seconds are < 10 add a 0 before
-            const time = `${('0' + ts.minutes).slice(-2)}:${('0' + ts.seconds).slice(-2)}`;
+            const hours = ('0' + Math.floor(ts.minutes / 60)).toString().slice(-2);
+            const minutes = ('0' + ts.minutes % 60).toString().slice(-2);
+            const seconds = ('0' + ts.seconds).toString().slice(-2);
```

Comment:
As we're doing 3 times the same thing here, can you create a function into `frontend/src/app/shared/helpers/time.helper.ts` (create the file)
`function getFormatedTime(): { hours: string, minutes: string, seconds: string };`

and then use it like that : 

`const { hours, minutes, seconds } = getFormatedTime();`